scripts/update_meilisearch_version.sh

```diff
@@ -104,12 +104,21 @@ check_last_exit_status() {
     fi
 }
 
+#Check if the Api key is available is the environment variable
+check_api_key() {
+    if [ -z $MEILISEARCH_MASTER_KEY ]; then
+        echo "${WARNING_LABEL}MEILISEARCH_MASTER_KEY is not available in the environment variables if you have an api key you must set it with the following command\nexport MEILISEARCH_MASTER_KEY=YOUR_API_KEY"
+    fi
+}
```

Comment:
Should we `exit` the script if it fails?

Reply:
```suggestion
check_api_key() {
    if [ -z $MEILISEARCH_MASTER_KEY ]; then
        echo "${WARNING_LABEL}MEILISEARCH_MASTER_KEY is not available in the environment variables. If you have an API key, set it with the following command 'export MEILISEARCH_MASTER_KEY=YOUR_API_KEY'"
    fi
}
```

Reply:
>Should we exit the script if it fails?

But what happens if you don't set a `MEILISEARCH_MASTER_KEY`? You can if you run your instance in `dev` mode

Reply:
You are right! My bad :)